meszstack.py

```diff
@@ -35,7 +35,7 @@ def end():
         yield meszstack
 
 
-tab = [1, 2, 3]
-
-for s in meszgen(tab):
-    print(s)
+# tab = [1, 2, 3]
+#
+# for s in meszgen(tab):
+#     print(s)
```

Comment:
Niech zostanie miejsce na jakieś testy
```suggestion
if __name__ == "__main__":
  tab = [1, 2, 3]
  for s in meszgen(tab):
      print(s)
```